sql/planbuilder/ddl.go

```diff
@@ -1035,6 +1040,21 @@ func (b *Builder) tableSpecToSchema(inScope, outScope *scope, db sql.Database, t
 
 	for i, def := range defaults {
```

Comment:
One thing that still bothers me since the refactor is that resolving defaults here is not sufficient. I had to resolve them after this step separately. I am not sure whether the defaults code here does anything, or if specific queries still need this logic, why and whether they should just use the other resolving code.

Reply:
The issue is that they can contain expressions with functions etc. So you would need to inline much more analysis into parsing for that to work. Not a bad idea, agree it's kind of half-done at this point.